src/app/home-page/components/finish-modal/finish-modal.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component,OnInit } from '@angular/core';
+import { TrackerService } from '../../../shared-module/tracker/tracker.service';
+import { TrackerEventEnum } from '../../../shared-module/tracker/types/tracker-event.enum';
+
+@Component({
+  selector: 'app-finish-modal',
+  templateUrl: './finish-modal.component.html',
+  styleUrls: ['./finish-modal.component.scss']
+})
+
+export class FinishModalComponent implements OnInit{
+  visibility:boolean = true;
+  
+  constructor(
+    private tracker: TrackerService
+  ) {}
+
+  ngOnInit(): void
+  {
+    this.tracker.event.subscribe((event)=> {
+      if (event.eventType === TrackerEventEnum.finish)
+      {
+        this.visibility = false;
+        setTimeout(()=>this.visibility = true,7000)
+      }
+      else
+      {
+        this.visibility = true;
+      }
+    });
```

Comment:
The modal doesn't appear after pomodoro finishes. Maybe I do something wrong. Could you please tell me what steps I should take to see it?